workspace/apps/pidp/src/app/features/profile/pages/personal-information/personal-information-resource.service.ts

```diff
@@ -11,7 +11,9 @@ import { ToastService } from '@core/services/toast.service';
 
 import { PersonalInformation } from './personal-information.model';
 
-@Injectable()
+@Injectable({
+  providedIn: 'root',
+})
```

Comment:
Unrelated change?

Reply:
Need to provide the service in `root` to use it outside of its module.